assignments/function-conversion.js

```diff
@@ -4,38 +4,44 @@
 // console.log("Function was invoked!");
 // };
   
-const myFunction = () => {
-    return 'Function was Invoked';
+let myFunction = () => {
+    console.log('Function was Invoked');
 }
-console.log(myFunction());
 
+myFunction();
 
-let anotherFunction = (param) => {
-   return param;
- };
 
-const anotherFunction = () => {
-    
-}
-console.log(anotherFunction());
+let anotherFunction = param => param;
+
+console.log(anotherFunction('Example!'));
+
+//with one parameter- we dont have to include the parentheses or the brackets around it 
+
 
  let add = (param1, param2) => {
    return param1 + param2;
-  };
-  add(1,2);
+  }
```

Comment:
You can make this a one-liner as the return statement and the curly braces are not required if the arrow function contains one statement/expression

Get more details [here](https://javascript.info/arrow-functions-basics)